interactive-clap-derive/src/derives/interactive_clap/mod.rs

```diff
@@ -373,36 +192,65 @@ pub fn impl_interactive_clap(ast: &syn::DeriveInput) -> TokenStream {
     }
 }
 
-fn context_scope_for_struct(
-    name: &syn::Ident,
-    context_scope_fields: Vec<proc_macro2::TokenStream>,
-) -> proc_macro2::TokenStream {
-    let interactive_clap_context_scope_for_struct = syn::Ident::new(
-        &format!("InteractiveClapContextScopeFor{}", &name),
-        Span::call_site(),
-    );
-    quote! {
-        pub struct #interactive_clap_context_scope_for_struct {
-            #(#context_scope_fields,)*
-        }
-        impl interactive_clap::ToInteractiveClapContextScope for #name {
-            type InteractiveClapContextScope = #interactive_clap_context_scope_for_struct;
-        }
-    }
-}
+/// these are common methods, reused for both the [structs] and `enums` derives
+pub(super) mod common_methods;
```

Comment:
should be at the top of the file

Reply:
resolved by https://github.com/near-cli-rs/interactive-clap/pull/26/commits/aad7bda7a14aeae49fe5a0407e879f8e08e0719b